api/net/Areas/Editor/Controllers/StorageController.cs

```diff
@@ -191,68 +191,50 @@ public IActionResult Move(string path, string destination, [FromRoute] string? l
     [SwaggerOperation(Tags = new[] { "Storage" })]
     public IActionResult Delete(string path, [FromRoute] string? location = "capture")
     {
-        var safePath = System.IO.Path.Combine(_options.GetRootPath(location), path.MakeRelativePath());
+        var safePath = Path.Combine(_options.GetRootPath(location), path.MakeRelativePath());
         if (!safePath.FileExists() && !safePath.DirectoryExists()) throw new InvalidOperationException($"File/folder does not exist: '{path}'");
 
-        var info = new ItemModel(safePath);
+        var item = new ItemModel(safePath);
         System.IO.File.Delete(safePath);
-        return new JsonResult(info);
+        return new JsonResult(item);
     }
 
     /// <summary>
     /// Make a clip from the target file identified in the clip parameter 'prefix'.
     /// </summary>
-    /// <param name="fileName"></param>
-    /// <param name="directory"></param>
+    /// <param name="path"></param>
     /// <param name="start"></param>
     /// <param name="end"></param>
-    /// <param name="clipNbr"></param>
-    /// <param name="prefix"></param>
+    /// <param name="outputName"></param>
     /// <returns></returns>
-    [HttpGet("clip")]
+    [HttpPost("clip")]
```

Comment:
HTTP GET shouldn't ever be used for introducing any type of 'change'.  We're creating a new file, it should be a POST.